scrapers/SexMex.yml

```diff
@@ -0,0 +1,51 @@
+name: "SexMex"
+
+sceneByURL: &byUrl
+  - action: scrapeXPath
+    url:
+      - sexmex.xxx
+    scraper: sceneScraper
+galleryByURL: *byUrl
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: &title //div[@class="panel-body"]/h4//text()
+      Details: &details //meta[@name="description"]/@content
```

Comment:
This gets only the synopsis  that is displayed in the main scenes page 
The full description seems to be ok with something like this 
```yaml
      Details: &details //div[@class="panel-body"]/p[not(@class)]
```

Reply:
Yeah that's true. I only looked up scenes with such short descriptions that didn't notice this.